Controller/DefaultController.php

```diff
@@ -99,13 +100,13 @@ protected function getResque()
      *
      * @return array
      */
-    private function getShowParameters()
+    private function getShowParameters(Request $request)
```

Comment:
Appears this is good on 2.0 branch of Symfony and up.

https://github.com/symfony/symfony/blob/2.0/src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php#L115